docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,559 @@
+ value: "camunda-zeebe-0.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-1.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-2.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-3.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-0.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-1.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-2.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-3.camunda-zeebe.camunda-paris.svc.cluster.local:26502"
+```
+
+The zeebe cluster size is 8 with a dual-region setup, resulting in 4 brokers per region.
+
+The address forms as follows:
+
+```bash
+$pod.$service.$namespace.svc.cluster.local:26502
+```
+
+- `$pod` = `$release`-zeebe-`$number`
+  - `$release` is the helm release name that you use. In our case, we're going with `camunda`.
+  - `$number` stems from zeebe being a statefulset, so it starts from 0 to 3 due to 4 replicas per region.
+  - resulting in `camunda-zeebe-(0-4)`
+- `$service` = `$release-zeebe`
+  - `$release` is the helm release name that you use. In our case, we're going with `camunda`.
+  - resulting in `camunda-release`
+- `$namespace` is the previously created namespace, in our case `camunda-london` and `camunda-paris`.
+- 26502 is the port for the initial contact at zeebe.
+
+In case you are going with a different helm release name than `camunda`, you must adjust the URLs for the ElasticSearch exporters `ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION0_ARGS_URL` and `ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION1_ARGS_URL`. Where `camunda` must be replaced with the release name that you have chosen.
+
+<details>
+  <summary>base-values.yml</summary>
+  <summary>
+
+```yml reference title="base-values.yml"
+https://github.com/camunda/c8-multi-region/blob/aws-operational/test/resources/aws/2-region/kubernetes/camunda-values.yml
+```
+
+  </summary>
+</details>
+
+#### Region0 values.yml
+
+The vital part of the overlay is that the `regionId` is set to 0. The rest is being handled in the base layer.
+
+```yaml reference title="region0-values.yml"
+https://github.com/camunda/c8-multi-region/blob/aws-operational/test/resources/aws/2-region/kubernetes/region0/camunda-values.yml
+```
+
+#### Region1 values.yml
+
+The vital part of the overlay is that the `regionId` is set to 1. The rest is being handled in the base layer.
+
+```yaml reference title="region1-values.yml"
+https://github.com/camunda/c8-multi-region/blob/aws-operational/test/resources/aws/2-region/kubernetes/region1/camunda-values.yml
+```
+
+</TabItem>
+<TabItem value="deploy" label="Deploy Camunda 8">
+
+The last step is to deploy Camunda 8 to both its regions, for which we chose London as region 0 and Paris as region 1. As mentioned, you may choose a different release name and should ensure you have adjusted your `base-values.yml` to consider this.
+
+```bash
+helm install camunda camunda/camunda-platform \
```

Comment:
we can pin it to 9.x but should generally work with any version moving forward.

Reply:
should we give version recommendations?
Starting 8.3.x, 8.4.x, 9.x?

Reply:
maybe something to check with the Distribution team on how to refer to a Helm chart for 8.5.x stack. For now guess we can just say whatever Helm chart version will be there with 8.5.0, wdyt?

Reply:
8.5.x stack --> 9x+

Reply:
ah you mean that, since it's already supported why not simply try to give a version recommendation?
We can go with the current 9.2 but maybe should have a matrix from when on it's supported.
8.3.x, 8.4.x, 9.x.y

Reply:
As we're not going to publicly advertise 8.3/8.4 support, let's use the Helm chart that refers to Camunda 8.5 y default (not sure which one will that be, but at least 9.x I guess)